ext/node/ops/ipc.rs

```diff
@@ -53,25 +163,66 @@ mod impl_ {
       Some(child_pipe_fd) => child_pipe_fd.0,
       None => return Ok(None),
     };
-
+    let ref_tracker = IpcRefTracker::new(state.external_ops_tracker.clone());
```

Comment:
Not for this PR, but I think we should move most of this module to `runtime/ops` or even create a separate `deno_ipc` crate. Besides the serialization code it seems abstract enough that we could expose an IPC API for `Deno.Command` API.